src/serializer/modules.js

```diff
@@ -97,8 +91,6 @@ class ModuleTracer extends Tracer {
         return undefined;
       } else {
         let result;
-        let oldErrorHandler = realm.errorHandler;
-        realm.errorHandler = () => "Fail";
```

Comment:
By deleting this code, the messages will appear on the screen as errors, not differentiating to our confused users that this is in case recoverable by delaying the require() call?